wiki/Tournaments/JHC/JHC_2024/en.md

```diff
@@ -0,0 +1,213 @@
+2. To ensure valid registrations, every participant will be sent to the [tournament screening](/wiki/Tournaments/Official_support#tournament-screening).
+   - Players must not have violated the [osu! community rules](/wiki/Rules) within the last 12 months.
+3. A list of all successfully registered players will be published after the registration.
+4. [Tournament staff members](/wiki/Tournaments/Official_support#staff) must not participate as players in the tournament.
+
+### Qualifier instructions
+
+1. In the Qualifier stage, all teams will play a specific pool designed by the mappool selectors.
+2. The Qualifier pool contains 6 maps, all of which will use [Free Mod](/wiki/Gameplay/Game_modifier#freemod) rules. The types of each maps are:
+   - DS | Dense Chordjack
+   - MD | Middle Chordjack
+   - LT | Light Chordjack
+   - RM | Rhythm Jack
+   - HV | Heavy Jack (Anchor Jack & Full Jack)
+   - SP | Speed Jack (Quadstream & Minijack)
+3. Players will play the mappool at least once (and up to twice) at a designated time. Their best combined score will be the final result.
+4. The mappool will be played in the following order: DS, MD, LT, RM, HV, SP. Players may not ask to play the pool in any other order.
+   - Teams may skip playing any beatmap on their second run.
+5. For each map, each team must have **only 1 member** to play. They may be exchanged freely after a map is concluded.
+6. An optional 5-minute break will be offered between the first and second playthrough of the mappool.
+7. All teams will play their qualifiers in separate rooms.
+   - It is *suggested* that teams do not broadcast or share their results publicly to avoid seed manipulation.
+8. The top 32 seeded teams will advance to the Round of 32, and \#33 ~ \#48 seeded teams will advance to the Loser Round 1.
+
+### Win conditions
+
+- In the Qualifiers, teams need to place in the top 48 seeded teams in order to advance to the double-elimination stage.
+- In the Round of 32 and Round of 16, teams need to win 5 maps to win a match (best of 9).
+- In the Quarterfinals, Semifinals and Finals, teams need to win 6 maps to win a match (best of 11).
+- In the Grand Finals, teams need to win 7 maps to win the match (best of 13).
+
+### Match instructions
+
+1. A referee will create a multiplayer room 15 minutes before the scheduled match time. Teams must join the lobby in that period.
+   - The room settings are `Game mode: "osu!mania"`, `Team mode: "Team Vs"` and `Score mode: "ScoreV2"`. The room name must follow the pattern of `JHC2024: ({Red Team}) VS ({Blue Team})`. The team mentioned first in the room name must be the red team, the team mentioned second in the room name must be the blue team.
+   - If the team name is so long or has illegal characters that the room can't be created, it may be considered to shorten or change the team name.
```

Comment:
```suggestion
   - If the team name is too long or contains illegal characters such that the room cannot be created, it may be considered to shorten or change the team name.
```